### PR TITLE
Cherry-pick PR:#2469

### PR DESCRIPTION
**Pull request by camilamacedo86**
Cherry-pick PR:https://github.com/operator-framework/operator-sdk/pull/2469

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,8 @@
-## Unreleased
-
-### Added
+## v0.15.2
```
Comment (joelanford) [inline CHANGELOG.md]: We'll change this later in the release PR.
```suggestion
## Unreleased
```

File: internal/scaffold/ansible/entrypoint.go [hunk 2]
```diff
@@ -36,14 +36,6 @@ func (e *Entrypoint) GetInput() (input.Input, error) {
...
-# https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html#openshift-specific-guidelines
-
-if ! whoami &>/dev/null; then
-  if [ -w /etc/passwd ]; then
-    echo "${USER_NAME:-runner}:x:$(id -u):$(id -g):${USER_NAME:-runner} user:${HOME}:/sbin/nologin" >> /etc/passwd
-  fi
-fi
-
+cd $HOME
 exec ${OPERATOR} exec-entrypoint ansible --watches-file=/opt/ansible/watches.yaml $@
```
Comment (camilamacedo86) [inline internal/scaffold/ansible/entrypoint.go]:
```suggestion
exec ${OPERATOR} exec-entrypoint helm --watches-file=$HOME/watches.yaml $@
```

File: doc/proposals/kubernetes-1.17.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ creation-date: 2019-12-19
 last-updated: 2019-12-19
 status: implementable
 see-also:
-  - https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.17.md#changes
```
Comment (joelanford) [inline doc/proposals/kubernetes-1.17.md]: We probably should put this fix in a separate PR (which will have to merge first) in case we have to revert the /etc/passwd change for some reason.

File: internal/scaffold/ansible/entrypoint.go [hunk 4]
```diff
@@ -36,14 +36,6 @@ func (e *Entrypoint) GetInput() (input.Input, error) {
...
-# This is documented here:
-# https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html#openshift-specific-guidelines
-
-if ! whoami &>/dev/null; then
-  if [ -w /etc/passwd ]; then
-    echo "${USER_NAME:-runner}:x:$(id -u):$(id -g):${USER_NAME:-runner} user:${HOME}:/sbin/nologin" >> /etc/passwd
-  fi
-fi
-
+cd $HOME
```
Comment (joelanford) [inline internal/scaffold/ansible/entrypoint.go]: I don't think this change should be here right?

File: doc/proposals/kubernetes-1.17.md (hunk 3 above)
Comment (camilamacedo86) [reply doc/proposals/kubernetes-1.17.md]: OKs.

**Comment by camilamacedo86** [comment]
ignoree
